int64 id
geometry_msgs/Point start
geometry_msgs/Point goal
float64 radius
float64 speed
uint32 agentMaxNum
float64 timeBoundary
float64 sightRadius
float64 timeBoundaryObst
geometry_msgs/Polygon[] Obstacles
---
float64 delta
